spec/factories/event_schedule.rb

```diff
@@ -0,0 +1,22 @@
+# Read about factories at https://github.com/thoughtbot/factory_girl
+
+FactoryGirl.define do
+  factory :event_schedule do
+    event
+    after(:build) do |event_schedule|
```

Comment:
There is a simple rule to follow when you create a factory. Only create what is really necessary and do not try to cover all circumstances at once. If some test cares about some specific aspects they should take care of setting those.